### ISSUE TITLE
An option to remember the reading point

### ISSUE THREAD

**Issue by hsicilia** [cheeaun/phanpy#269]
I don't know if it is possible to save in the configuration the most recent tooth read when we close the browser tab.

That way, when we log in again later, it would position us on that tooth and we could read everything we have missed.

**Comment by zefhemel** [comment]
This is a duplicate of https://github.com/cheeaun/phanpy/issues/176 no?

**Comment by hsicilia** [comment]
#176 is more than what I ask for :D

I was only thinking of it working locally in a browser.

But from reading this issue I see that some of the problems would be the same.

I use Fedilab in my mobile and it has this option, and it is very useful for me.

**Comment by dermoth** [comment]
Agreed, the app should allow remembering the last read position, locally. In particular when I return to my home screen using the back key (on mobile, installed as an app) the page refreshes upon reopening the app and I lose my position.

**Comment by amdavidson** [comment]
This is also my challenge with using phanpy. 

I like it quite a bit but losing my spot as soon as new toots are loaded makes it unusable (for me and the way I use mastodon).

**Comment by earth2marsh** [comment]
I love using Phanpy on my Android phone, but often when I leave the progressive web app and come back to it, it will reload, losing my place. This is particularly frustrating when I open a link in Chrome that was shared on Mastodon, and then come back to find that I've lost my place.

I'd be happy if Phanpy just stored my last read message and jumped me to that message in my timeline.

Beyond that, being able to track my most recently read message across devices (similar to how how Kindle readers can "jump to last read") might be cool. It might also be necessary to change the way loading works so that it doesn't always jump you to the newest message.

**Comment by brihuega** [comment]
> I love using Phanpy on my Android phone, but often when I leave the progressive web app and come back to it, it will reload, losing my place. This is particularly frustrating when I open a link in Chrome that was shared on Mastodon, and then come back to find that I've lost my place.
> 
> I'd be happy if Phanpy just stored my last read message and jumped me to that message in my timeline.
> 
> Beyond that, being able to track my most recently read message across devices (similar to how how Kindle readers can "jump to last read") might be cool. It might also be necessary to change the way loading works so that it doesn't always jump you to the newest message.

I have the same experience. I vote for this enhancement.
